integration-cli/cli/build/build.go

```diff
@@ -0,0 +1,30 @@
+package build
+
+import (
+	"strings"
+
+	icmd "github.com/docker/docker/pkg/testutil/cmd"
+)
+
+// WithDockerfile specify a dockerfile string to be used as Dockerfile for build operations
```

Comment:
If we have a type (see below), the description could become a bit clearer; `WithDockerfile creates / returns a CmdOperator to set the Dockerfile for a build operation` (or something like that)